src/main/java/com/jaoafa/mymaid4/command/Cmd_Calctree.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * jaoLicense
+ *
+ * Copyright (c) 2021 jao Minecraft Server
+ *
+ * The following license applies to this project: jaoLicense
+ *
+ * Japanese: https://github.com/jaoafa/jao-Minecraft-Server/blob/master/jaoLICENSE.md
+ * English: https://github.com/jaoafa/jao-Minecraft-Server/blob/master/jaoLICENSE-en.md
+ */
+
+package com.jaoafa.mymaid4.command;
+
+import cloud.commandframework.Command;
+import cloud.commandframework.arguments.standard.BooleanArgument;
+import cloud.commandframework.context.CommandContext;
+import cloud.commandframework.meta.CommandMeta;
+import com.jaoafa.mymaid4.lib.CommandPremise;
+import com.jaoafa.mymaid4.lib.MyMaidCommand;
+import com.jaoafa.mymaid4.lib.MyMaidLibrary;
+import com.sk89q.worldedit.IncompleteRegionException;
+import com.sk89q.worldedit.bukkit.WorldEditPlugin;
+import com.sk89q.worldedit.regions.Region;
+import com.sk89q.worldedit.world.World;
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.format.NamedTextColor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Cmd_Calctree extends MyMaidLibrary implements CommandPremise {
+    @Override
+    public MyMaidCommand.Detail details() {
+        return new MyMaidCommand.Detail(
+            "calctree",
+            "WorldEditの選択範囲で植木算を行います。"
+        );
+    }
+
+    @Override
+    public MyMaidCommand.Cmd register(Command.Builder<CommandSender> builder) {
+        return new MyMaidCommand.Cmd(
+            builder
+                .meta(CommandMeta.DESCRIPTION, "WorldEditの選択範囲で植木算を行います。")
+                .argument(BooleanArgument.optional("placeEdgeTree", true))
```

Comment:
```suggestion
                .argument(BooleanArgument.optional("placeEdgeTree", true), ArgumentDescription.of("説明文"))
```
のように説明をつけるのが望ましいかもしれません（`/mymaidhelp` とかでヘルプが出るようになったので）